src/util/storageVolume.tsx

```diff
@@ -86,16 +86,64 @@ export const getLxdDefault = (
 };
 
 export const volumeTypeForDisplay = (volume: LxdStorageVolume) => {
-  const typePrefix =
+  const volumeDisplayName =
     volume.type === "virtual-machine"
       ? "VM"
       : capitalizeFirstLetter(volume.type);
-  const typeSuffix = volume.name.includes("/") ? " (snapshot)" : "";
-  return `${typePrefix}${typeSuffix}`;
+  return volumeDisplayName;
 };
 
 export const contentTypeForDisplay = (volume: LxdStorageVolume) => {
   return volume.content_type === "iso"
     ? "ISO"
     : capitalizeFirstLetter(volume.content_type);
 };
+
+export const generateLinkForVolumeDetail = (args: {
```

Comment:
shouldn't this be located in the StorageVolumeNameLink component? Though not sure what's best here.

Reply:
there is a bit of logic in this function and it's just a preference to keep logic away from components whenever I can. I'm not too strongly opinionated about this though, if you feel it's more consistent with rest of the code to move logic into the component then that's also fine?

Reply:
just moved the function into the `StorageVolumeNameLink` component, still a function, that makes sense.

Reply:
We don't really have a tight rule around this. Some functions are in module specific files under util -- you probably have seen it already. When a function is component-specific and not likely to be reused, I tend to keep functions local with the component.